### DISCUSSION TITLE
Beginners and their Mistakes

### DISCUSSION THREAD

**Discussion by smooth-web** [community/community#106702] [New to GitHub] [answered]
What is the best approach for newbies, when we are starting out new activities. What's the best research material for us??

**Comment by gil-son** [comment, marked as answer]
Hello,

- Create and develop good habits
- Manage yourself discipline
- Create goals and plans
- Remember to be patient and respect yourself
- Avoid to compare with others

So, try to identify your focus and after that choice the road or create one

On that page you can find some examples of [Roadmap](https://roadmap.sh/)

**Comment by rafidRD** [comment]
You will find thousands of tutorial and posts on the internet. 
Just google it. You won't be able to do it untill you practically do it. 
[reference](https://github.com/github/roadmap?tab=readme-ov-file)

**👇🏼you will find an accept button here, accept it**
